### DISCUSSION TITLE
Ready for full planet?

### DISCUSSION THREAD

**Discussion by hyperknot** [systemed/tilemaker#619] [General]
I've been following the developments here and I see that there is great progress towards full planet generation.

How is the state currently for doing a full planet based on OMT schema? 

Issues detailing broken geometries:
https://github.com/systemed/tilemaker/discussions/434
https://github.com/systemed/tilemaker/issues/315

Memory usage improvements: 
https://github.com/systemed/tilemaker/pull/612
https://github.com/systemed/tilemaker/pull/499
https://github.com/systemed/tilemaker/pull/618 - still open

I'd like to run this on a 128 GB server. If I understand with the recent memory optimisation PRs, this should be no problem. I'm more concerned about the broken geometries. Are they mostly fixed by now?

Planetiler has missing tiles in full planet run, so until that is fixed it's not a viable alternative:
https://github.com/onthegomap/planetiler/issues/760

**Comment by systemed** [comment]
As best as I know, the broken geometries are fixed. I've recently run tilemaker over Europe and checked several previously problematic locations, and they all seem good.

128GB should be absolutely no problem using the standard `--store` option to point to somewhere on SSD. With #618 you should be able to do it in ~24GB if you also specify `--shard-stores`.

**Reply by hyperknot** [reply]
Thank you for the reply. I'll try to make a planet run.
